experimental/Schemes/src/ToricIdealSheaves/constructors.jl

```diff
@@ -159,7 +159,7 @@ function ideal_sheaf(X::NormalToricVariety, tau::Cone)
   # Maybe it's a mistake in the book and we really need the dual?
   #tau_perp = polarize(tau)
   ideal_dict = IdDict{AbsAffineScheme, Ideal}()
-  # We are using Equation (3.2.7) in CLS to determine the local 
+  # We are using Equation (3.2.7) in [CLS11](@cite) to determine the local
```

Comment:
It does not make sense to input `@cite` here since this is just a comment in function and nothing that is rendered in the documentation. It is for developers and developers of the toric geometry part know what CLS refers to.